SepulchureSaga/04ShadowfallRise.cs

```diff
@@ -0,0 +1,140 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreDailies.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/CoreAdvanced.cs
+using RBot;
+
+public class ShadowfallRise
+{
+    public ScriptInterface Bot => ScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreFarms Farm = new CoreFarms();
+    public CoreAdvanced Adv = new CoreAdvanced();
+    public CoreStory Story = new CoreStory();
+    public CoreDailies Daily = new();
+
+    public void ScriptMain(ScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        StoryLine();
+
+        Core.SetOptions(false);
+    }
+
+    public void StoryLine()
+    {
+        if (Core.isCompletedBefore(6592))
+        {
+            
+            Core.Logger("You have already completed ShadowfallRise Storyline");
+            return;
+        }
+
+        Story.PreLoad();
+
+        Core.EquipClass(ClassType.Solo);
+
+        //Fluid Dynamics 6539
+        Story.KillQuest(6539, "noxustower", "Slimeskull");
+
+        //M-ossification 6540
+        Story.KillQuest(6540, "noxustower", "Doomwood Treeant");
+
+        //A-gore-able 6541
+        Story.KillQuest(6541, "noxustower", "Sanguine Souleater");
+
+        //Raw-r 6542
+        Story.KillQuest(6542, "noxustower", "Lightguard Caster");
+
+        //Light 'em Up 6543
+        Story.KillQuest(6543, "noxustower", "Lightguard Caster");
+
+        //Spellbound 6544
+        Story.MapItemQuest(6544, "noxustower", 6018, 4);
+        Story.KillQuest(6544, "noxustower", "Lightguard Caster");
```

Comment:
not that i think it matters to much but please put mapitemquests after killquests 🤷